dev-docs/modules/consentManagement.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: page
+title: Module - GDPR ConsentManagement
```

Comment:
@mjacobsonny 
What do you want to call the module? GDPR Consent Management or just "Consent Management"